deluge/core/core.py

```diff
@@ -770,9 +776,9 @@ def get_torrent_status(
             all_keys=not keys,
         )
 
-    @export
+    @export  # type: ignore
     @defer.inlineCallbacks
-    def get_torrents_status(
+    def get_torrents_status(  # type: ignore
```

Comment:
Ignored a bunch of stuff on inlineCallbacks for now. The generator would have to be properly type hinted as a generator, which is weird. This is one thing that would be a _lot_ easier with the `maybe_coroutine` branch. Type hinting an async function is the same as doing a normal function, no need to do fancy gen typing stuff.